### ISSUE TITLE
Documentation for installing on Windows

### ISSUE THREAD

**Issue by Aralox** [roflcoopter/viseron#838]
Could you please add an example command and advice for installing Viseron on Windows 10?
There are only linux examples on https://viseron.netlify.app/docs/documentation/installation#running-viseron.

Context: I have Docker desktop installed with WSL 2, and an Nvidia GPU for CUDA (https://docs.docker.com/desktop/features/gpu/). I'd like to move away from Frigate to Viseron to take advantage of my GPU. 

Thank you for your work!

EDIT: I was unclear above - I have docker desktop installed as a windows application, and also have WSL 2 installed (in order to do the GPU stuff, as that's what docker says it needs in the link above).

**Comment by Aralox** [comment]
Sorry I was unclear - I have docker desktop installed as a windows application, not in WSL. I installed WSL just so that docker can use my GPU - that's one of the requisites it mentioned in https://docs.docker.com/desktop/features/gpu/.
I did try the linux command (just for fun, I knew it wouldn't work on windows anyway as it was looking for /etc/localtime).
I hope the above makes sense. I'm looking to run this container on a windows machine, from a windows command line (or via the docker UI / configuration pages, with some way to just make it run on computer boot). Thank you :)

**Comment by roflcoopter** [comment]
I see.
Sadly i dont have a machine to test this so you are gonna have to do some digging on your own.

Seems to be possible tho: https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/linux-containers

But why cant you run Viseron in WSL instead?

**Comment by Aralox** [comment]
I finally got some time to give this another go - you're right, it's possible, and actually docker desktop for windows automatically integrates with WSL 2 if it's present. That allows calling docker commands in the WSL2 linux environment, which means I can use your example configurations. I've managed to get a container with Viseron going.
I'm having trouble actually picking up my camera stream, and it looks like it's not talking to my GPU yet, but I'll raise separate issues for that. Thanks!
